packages/eslint-plugin/tests/rules/member-ordering-natural-case-insensitive-order.test.ts

```diff
@@ -0,0 +1,135 @@
+import rule from '../../src/rules/member-ordering';
```

Comment:
for organisational sake - could we put the tests within a subfolder?
`packages/eslint-plugin/tests/rules/member-ordering/member-ordering-natural-case-insensitive-order.test.ts`